engine/framework/model/master.go

```diff
@@ -25,32 +25,23 @@ import (
 
 type (
 	// MasterStatusCode is used in framework to manage job status
-	MasterStatusCode int32
-	// WorkerType represents task type, such as DM worker, DM master, etc.
-	WorkerType int64
+	MasterStatusCode int8
 )
 
-// MasterUpdateColumns is used in gorm update
-// TODO: using reflect to generate it more generally
-// related to some implement of gorm
-var MasterUpdateColumns = []string{
-	"updated_at",
-	"project_id",
-	"id",
-	"type",
-	"status",
-	"node_id",
-	"address",
-	"epoch",
-	"config",
-}
+// Job master statuses
+const (
```

Comment:
Can we hard code the statuses here instead of using iota? Adding or removing might cause confusion when using iota.

Reply:
I think it's a convention for the use of `iota`.
What about `Adding comment for the MasterStatusCode in DB`?

Reply:
Hard code may be better.